Automattic/wp-calypso
# Jetpack Cloud: Move Cloud-specific code to its own directory inside client/

Jetpack Cloud used to have its own separate entry point in Webpack, but has since been moved into the main Calypso entry point. This PR moves all remaining Jetpack Cloud-specific code out of `client/landing/jetpack-cloud` to `client/jetpack-cloud`, since `client/landing` is conventionally a place for apps that live independently in separate entry points (complete with their own Redux stores, etc.).

#### Changes proposed in this Pull Request

* Move Jetpack Cloud-specific code from `/client/landing/jetpack-cloud` to `/client/jetpack-cloud`.
* Fix linter warnings by refactoring `getJetpackAdminUrl` in `client/components/jetpack/sidebar/index.jsx` so that it references `lib/url` instead of `url`.

#### Testing instructions

* Verify that `yarn start-jetpack-cloud` still compiles and runs Jetpack Cloud.
* Verify that all Jetpack Cloud-specific pages work as they do in production.
* Verify that all Jetpack Cloud styles still appear and behave as they do in production.
* **Specifically:** Verify that the **WP-Admin** sidebar link in Jetpack Cloud correctly links to the selected site's Jetpack dashboard.
